src/clock.js

```diff
@@ -0,0 +1,57 @@
+
+
+class TIME
```

Comment:
w nazwie klasy tylko pierwsza litera jest wielka 'Time'. Btw to lepiej chyba nazwać tą klase Timer